yandex_contest/binary_sum.py

```diff
@@ -1,44 +1,35 @@
-def convert(string):
-    list_1 = []
-    list_1[:0] = string
-    list_x = [int(i) for i in list_1]
+def convert_to_list(string):
+    list_x = [int(i) for i in string]
```

Comment:
Here you don't need to save this intermediate value, can just return it
```suggestion
    return [int(i) for i in string]
```